dev-python/pycollada/pycollada-0.6-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library for reading and writing COLLADA documents"
+HOMEPAGE="https://pycollada.readthedocs.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+RDEPEND="
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/python-dateutil-2.2[${PYTHON_USEDEP}]
+"
+DEPEND="
+	${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/python-dateutil[${PYTHON_USEDEP}] )
+"
+
+REQUIRED_USE="${PYTHON_REQ_USE}"
+
+DOCS=( AUTHORS.md CHANGELOG.rst COPYING README.markdown )
+
+src_compile() {
+	distutils-r1_src_compile
+
+	if use doc ; then
+		pushd docs > /dev/null || die
+		emake html
+		popd > /dev/null || die
+	fi
+}
+
+src_install() {
+	distutils-r1_src_install
+
+	use doc && local HTML_DOCS=( docs/_build/html/. )
+	einstalldocs
+
+	if use examples ; then
+		insinto /usr/share/${P}/
+		doins -r examples
+	fi
+
+	install_test_data() {
```

Comment:
This belongs in `python_install`.

Reply:
Shall we call this in all cases? I would rather go and wrap this function and the `python_foreach_impl` call in an `if use test` statement. What do you say?

Reply:
Probably yes. `USE=test` is for running tests, not installing them. I see no convincing case not to install those files.

Reply:
Ok, I understand, so I leave it like this.